### PR TITLE
Fix Issue #12: Incorrect Loading State

### PR DESCRIPTION
**Pull request by jilio**
**tldr:** Fix for https://github.com/nanostores/query/issues/12

In the latest release v0.1.0 `loading` state was being set to `false` in the following line: https://github.com/nanostores/query/blob/de0ca288c93b7b7467c80520188ad5b8649adac2/lib/main.ts#L178

This behavior contradicts the previous setup, where the loading state was set to `true`:
https://github.com/nanostores/query/blob/132186c57744150903871b6d38fa6bf161eb3078/lib/main.ts#L176

which led to an incorrect `loading` state as described here: https://github.com/nanostores/query/issues/12

This pull request rectifies this issue, ensuring that the loading state functions as intended.

### CODE REVIEW

**Comment by jilio** [comment]
Apparently, my fix doesn't work 😅
